templates/resources/common/jmx-exporter/jmx-exporter.yaml

```diff
@@ -0,0 +1,4 @@
+---
+hostPort: localhost:9011
```

Comment:
If you expose it on `localhost`, does it still listen on every network interface? Typically you would need to set it so `0.0.0.0:9011` I think...

Reply:
This is the configuration for jmx-exporter to connect to hostPort of local JMX. I added some comments to this yaml.

Reply:
👍